components/organisms/Feedback.js

```diff
@@ -0,0 +1,188 @@
+    const feedbackDropdown = document.getElementById("feedbackDropdown");
+    const caret = document.getElementById("caret");
+
+    feedbackDropdown.classList.toggle("active");
+    feedbackButton.getAttribute("aria-expanded") === "true"
+      ? feedbackButton.setAttribute("aria-expanded", false)
+      : feedbackButton.setAttribute("aria-expanded", true);
+
+    feedbackButton.getAttribute("aria-expanded") === "true"
+      ? (caret.innerHTML = "&#9660;")
+      : (caret.innerHTML = "&#9656;");
+  }
+
+  return (
+    <div
+      id="feedbackDropdown"
+      className="feedbackDropdown bg-custom-blue-blue"
+      data-testid="feedbackDropdown"
+    >
+      <div role="status">
+        {submitted ? (
+          <div className="layout-container text-white flex justify-between">
+            <h2 className="text-xs lg:text-sm font-body mt-2 mb-4">
+              {t("thankYouFeedback")}
+            </h2>
+            <button
+              id="feedbackClose"
+              onClick={closeFeedback}
+              className="font-body text-white flex text-left sm:ml-4 my-2 sm:my-0"
+              aria-haspopup="true"
+              aria-expanded="false"
+              aria-controls="feedbackDropdown"
+              data-testid="closeButton"
+            >
+              <span
+                id="close"
+                className="text-h3 lg:text-h2 leading-4 lg:leading-10"
+              >
+                &times;
+              </span>
+              <span className="text-xs leading-4 lg:text-sm underline ml-2 lg:leading-10">
+                {t("close")}
+              </span>
+            </button>
+          </div>
+        ) : (
+          ""
+        )}
+      </div>
+      {submitted ? (
+        ""
+      ) : (
+        <div className="layout-container text-white pb-4">
+          <div className="flex justify-between">
+            <h2 className="text-h4 lg:text-h3 lg:text-sm font-display mt-2 mb-4 w-48 sm:w-auto">
+              {t("improveService")}
+            </h2>
+            <button
+              id="feedbackClose"
+              onClick={closeFeedback}
+              className="font-body text-white flex text-left sm:ml-4 my-2 sm:my-0"
+              aria-haspopup="true"
+              aria-expanded="false"
+              aria-controls="feedbackDropdown"
+              data-testid="closeButton"
+            >
+              <span
+                id="close"
+                className="text-h3 lg:text-h2 leading-4 lg:leading-10"
+              >
+                &times;
+              </span>
+              <span className="text-xs leading-4 lg:text-sm underline ml-2 lg:leading-10">
+                {t("close")}
+              </span>
+            </button>
+          </div>
+          <ul className="list-outside list-disc px-6 py-2">
+            <li className="text-xs lg:text-sm font-body mb-4">
+              <b>{t("reportAProblemNoReply", { lng: props.language })}</b>{" "}
```

Comment:
We should usually pick `<strong>` instead of `<b>`, although the difference is subtle.

- If the bolded text is worth emphasizing (eg, "you really have to pay attention this"), then we should use `<strong>`
- If the bolded text is just for looking bold on the page to fit in with the design, then `<b>` is good.

In this case, I think we do want to emphasize that people won't receive a reply.

👍